nsv13/code/modules/turbolift/lazylift.dm

```diff
@@ -111,14 +111,25 @@ That's it, ok bye!
 	var/obj/machinery/lazylift/master/master = null
 	var/list/platform = list() //The """platform""" of the lift that's going to move up and down. This is just a list of turfs that we own.
 	var/list/doors = list()
-	var/static/list/moving_blacklist = list(/obj/machinery/lazylift, /obj/machinery/lazylift/master, /obj/machinery/light, /obj/structure/cable, /obj/machinery/power/apc, /obj/machinery/airalarm, /obj/machinery/firealarm, /obj/structure/grille, /obj/structure/window, /obj/machinery/camera)
 	var/id = null // A unique ID that links a specific lift to a specific button
+	var/static/list/moving_blacklist = typecacheof(list(
```

Comment:
I know it was already using one, but remind me why we want to use a typecache here instead of just finding a path in a flat list?

Reply:
it's faster

Reply:
-deep sigh-

Reply:
I don't know ma'am I just work here

Reply:
Yeah not your fault, just... Byond.